js/cards/disk2.ts

```diff
@@ -167,7 +167,7 @@ interface BaseDrive {
     /** Displayed disk name */
     name: string;
     /** (Optional) Disk side (Front/Back, A/B) */
-    side?: string;
+    side?: string | undefined;
```

Comment:
This confuses me; it looks like a mistake.

To me, it seems like `side` should be optional since it doesn't make sense for some formats (like hard drive images).